operators.py

```diff
@@ -103,6 +105,11 @@ def init_simulation(self):
 
         simulation_class = self.scene.elements_nodes[simulation_node.name]
 
+        # simulation_class.frame_start - not working! Why? To figure out.
+        self.frame_start = simulation_class.params['frame_start']
```

Comment:
Do you mean `simulation_class.frame_end` works but `simulation_class.frame_start` doesn't?

Reply:
Yes. To get the value of frame_start you need to write this way:
` self.frame_start = simulation_class.params['frame_start']`
Instead
`self.frame_start = simulation_class.frame_start`

Reply:
I found a mistake and fixed it. Now I will `push`.

Reply:
I have no clear idea, but it seems to me the only difference between `frame_end` and `frame_start` is that 
https://github.com/taichi-dev/taichi_elements/blob/bdada022532f1ae5a941c963d947687b609fc8e6/handlers.py#L125
sets `frame_end` but not `frame_start`. Do we also need to set `frame_start` here?

Reply:
fix: 9ae8d45380a174f3a767a719a3509fe8148cecc8